source/retryable-reads/tests/find-serverErrors.yml

```
@@ -0,0 +1,180 @@
+database_name: &database_name "retryable-reads-tests"
+collection_name: &collection_name "coll"
+
+data:
+    - {_id: 1, x: 11}
+    - {_id: 2, x: 22}
+    - {_id: 3, x: 33}
+    - {_id: 4, x: 44}
+    - {_id: 5, x: 55}
+
+minServerVersion: '4.1.6' # 4.2
+
+tests:
+    -
+        description: "Find succeeds after InterruptedAtShutdown"
+        clientOptions:
+            retryReads: true
+        failPoint: &failCommand_failPoint
+            configureFailPoint: failCommand
+            mode: { times: 1 }
+            data: { failCommands: [find], errorCode: 11600 }
+        operation: &retryable_operation
+            name: find
+            object: collection
+            arguments: { filter: {}, sort: { _id: 1 }, limit: 4, batchSize: 2 }
```

Remove `batchSize` because it causes extraneous getMores.
```suggestion
            arguments: { filter: {}, sort: { _id: 1 }, limit: 4 }
```